src/Featurizers/UnitTests/StandardScaleWrapperFeaturizer_UnitTest.cpp

```diff
@@ -0,0 +1,191 @@
+// ----------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License
+// ----------------------------------------------------------------------
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+
+#include "../../3rdParty/optional.h"
+#include "../../Featurizers/StandardScaleWrapperFeaturizer.h"
+#include "../TestHelpers.h"
+#include "../../Traits.h"
+
+namespace NS = Microsoft::Featurizer;
+
+// test for no valid input
+TEST_CASE("no valid input") {
```

Comment:
I would expect to see each of these tests repeated for every flag permutation:

true, true
true,  false
false, true
false, false